README.md

```diff
@@ -50,6 +52,14 @@ IronDB installation behind a load balancer.  For example,
         'http://host2:8112/graphite/1',
     )
 
+If you are pointing graphite at a Circonus SaaS account, set the token
+to a valid Circonus Auth Token and set the URL to the public API URL.
+
+    CIRCONUS_TOKEN = '<your-token-uuid>'
+    IRONDB_URLS = (
+        'https://api.circonus.com/graphite'
+    )
+
```

Comment:
Might be good to direct them to docs here about allow/deny semantics for apps in the API.